instrumentation/apache-httpclient/apache-httpclient-5.2/library/README.md

```diff
@@ -0,0 +1,60 @@
+# Library Instrumentation for Apache Http client version 5.2
+
+Provides OpenTelemetry instrumentation for [Apache Http Client 5.2](https://hc.apache.org/httpcomponents-client-5.2.x/).
+
+## Quickstart
+
+### Add these dependencies to your project
+
+Replace `OPENTELEMETRY_VERSION` with the [latest
+release](https://search.maven.org/search?q=g:io.opentelemetry.instrumentation%20AND%20a:opentelemetry-apache-httpclient-5.2).
+
+For Maven, add to your `pom.xml` dependencies:
+
+```xml
+<dependencies>
+  <dependency>
+    <groupId>io.opentelemetry.instrumentation</groupId>
+    <artifactId>opentelemetry-apache-httpclient-5.2</artifactId>
+    <version>OPENTELEMETRY_VERSION</version>
+  </dependency>
+</dependencies>
+```
+
+For Gradle, add to your dependencies:
+
+```groovy
+implementation("io.opentelemetry.instrumentation:opentelemetry-apache-httpclient-5.2:OPENTELEMETRY_VERSION")
+```
+
+### Usage
+
+The instrumentation library provides a builder class `ApacheHttpClient5Telemetry` that wraps
+an instance of the `HttpClientBuilder` to provide OpenTelemetry-based spans and context
+propagation.
+
+```java
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.instrumentation.apachehttpclient.v5_2.ApacheHttpClient5Telemetry;
+import org.apache.hc.client5.http.classic.HttpClient;
+import org.apache.hc.client5.http.impl.classic.HttpClientBuilder;
+
+public class ApacheHttpClient5Configuration {
+
+  private OpenTelemetry openTelemetry;
+
+  public ApacheHttpClient5Configuration(OpenTelemetry openTelemetry) {
+    this.openTelemetry = openTelemetry;
+  }
+
+  // your configuration of the HttpClient goes here:
```

Comment:
Imo this comment doesn't make sense. In the okhttp client doc where it is copies from the intention was to say that if you need to configure the http client then do it in this method but here this method does something completely different. perhaps something like `creates a new http client builder for constructing http clients with open telemetry instrumentation`